### PR TITLE
[IMP] marketing automation: updated testing_running for Odoo 16

### PR DESCRIPTION
**Pull request by ksc-odoo**
Project Task: https://www.odoo.com/web#id=3429774&cids=3&menu_id=4720&action=333&active_id=3835&model=project.task&view_type=form

### CODE REVIEW

File: content/applications/marketing/marketing_automation/getting_started/testing_running.rst [hunk 1]
```diff
@@ -2,33 +2,45 @@
...
 
-First, open the :menuselection:`Marketing Automation` application, and select the desired campaign,
-which reveals that campaign's detail form.
+Testing campaigns
+=================
 
-On the campaign detail form, make sure the campaign already has activities configured (or build a
-campaign by following the directions here on :doc:`this documentation
+To test a marketing campaign, open the :menuselection:`Marketing Automation` application, and select
+the desired campaign to test, which reveals that campaign's detail form.
```
Comment (tiku-odoo) [inline content/applications/marketing/marketing_automation/getting_started/testing_running.rst:13]:
```suggestion
To test a marketing campaign, open the :menuselection:`Marketing Automation` application, and select
the desired campaign to test, which reveals the campaign's detail form.
```

File: content/applications/marketing/marketing_automation/getting_started/testing_running.rst [hunk 2]
```diff
@@ -2,33 +2,45 @@
...
+.. image:: testing_running/launch-test-popup-window.png
+   :align: center
+   :alt: Launch a test pop-up window that appears in Odoo Markting Automation.
 
-When clicked, a :guilabel:`Launch a test` pop-up window appears. In the drop-down field, choose an
-existing record (or create a new one) to run the test on. To create a brand new record, click the
-:guilabel:`Search More...` link at the bottom of the drop-down menu, and then click the
-:guilabel:`Create` button.
+Click the field to reveal a drop-down menu of contacts. From this drop-down menu, select an existing
+contact (or create a new one) to run the test on.
```
Comment (tiku-odoo) [inline content/applications/marketing/marketing_automation/getting_started/testing_running.rst:33]:
```suggestion
Click the field to reveal a drop-down menu of contacts. From this drop-down menu, select an existing
contact (or create a new one) to run the test on.
```

Click what field? Can multiple be selected?

File: content/applications/marketing/marketing_automation/getting_started/testing_running.rst [hunk 3]
```diff
@@ -2,33 +2,45 @@
...
 
-Doing so reveals a blank contact form, in which the :guilabel:`Name` and :guilabel:`Email` **must**
-be entered. When all the necessary information has been entered, click :guilabel:`Save`, and Odoo
-returns to the :guilabel:`Launch a test` pop-up, with that new record in the :guilabel:`Choose an
-existing record or create a new one to test` field.
+To create a new contact directly from the :guilabel:`Launch a test` pop-up window, start typing the
+name of the new contact in the blank field, and click :guilabel:`Create and edit...`. Doing so
+reveals a blank :guilabel:`Create Record` pop-up form, in which the necessary contact information
+(:guilabel:`Email`, :guilabel:`Mobile`, etc.) *must* be entered, in order for the test to work.
```
Comment (tiku-odoo) [inline content/applications/marketing/marketing_automation/getting_started/testing_running.rst:38]:
```suggestion
To create a new contact directly from the :guilabel:`Launch a test` pop-up window, start typing the
name of the new contact in the blank field, and click :guilabel:`Create and edit...`. Doing so
reveals a blank :guilabel:`Create Record` pop-up form, in which the necessary contact information
(:guilabel:`Email`, :guilabel:`Mobile`, etc.) *must* be entered, in order for the test to work.
```

I would say this action deserves a screenshot and the screenshot in lines 22-24 or Lines 28-30 could be deleted.

File: content/applications/marketing/marketing_automation/getting_started/testing_running.rst [hunk 4]
```diff
@@ -2,33 +2,45 @@
...
 
-Once a record is selected, click :guilabel:`Continue`, and Odoo reveals the campaign test page.
+When all the necessary fields have been entered, click :guilabel:`Save & Close` to return to the
+:guilabel:`Launch a test` pop-up window.
+
+Once a contact is selected, click :guilabel:`Launch` to reveal the campaign test page.
 
 .. image:: testing_running/test-screen.png
    :align: center
```
Comment (tiku-odoo) [inline content/applications/marketing/marketing_automation/getting_started/testing_running.rst:46]: You have two very similar screenshots one after the other with not arrows or red-outlines. I found this to be confusing.  @lines 63-65

File: content/applications/marketing/marketing_automation/getting_started/testing_running.rst [hunk 5]
```diff
@@ -43,10 +55,83 @@ To start a test, click the :guilabel:`Run` button, represented by a :guilabel:`
...
+upper-right corner) moves to the :guilabel:`Completed` stage.
+
+To stop a test before all the workflow activities are completed, click the :guilabel:`Stop` button
+in the upper-left corner of the campaign test page.
+
+Running campaigns
+=================
+
+To run a campaign, navigate to :menuselection:`Marketing Automation app`, and select the desired
+campaign to run.
```
Comment (tiku-odoo) [inline content/applications/marketing/marketing_automation/getting_started/testing_running.rst:77]:
```suggestion
To run a campaign, navigate to :menuselection:`Marketing Automation app`, and select the desired
campaign to run.
```

You mention Marketing Automation Application at the start of the doc and now use a menu selection for just app here. May want to keep consistent.

File: content/applications/marketing/marketing_automation/getting_started/testing_running.rst [hunk 6]
```diff
@@ -43,10 +55,83 @@ To start a test, click the :guilabel:`Run` button, represented by a :guilabel:`
...
+
+On the campaign detail form, with all the desired activities ready in the :guilabel:`Workflow`
+section, click :guilabel:`Start` in the upper-right corner to officially run the campaign to the
+configured target audience specified on the campaign detail form.
+
+.. image:: testing_running/start-button-campaign-detail-form.png
+   :align: center
+   :alt: The Start button in the upper-left corner of the marketing campaign detail form.
```
Comment (tiku-odoo) [inline content/applications/marketing/marketing_automation/getting_started/testing_running.rst:85]: This image is not necessary.

File: content/applications/marketing/marketing_automation/getting_started/testing_running.rst [hunk 7]
```diff
@@ -43,10 +55,83 @@ To start a test, click the :guilabel:`Run` button, represented by a :guilabel:`
...
+   :guilabel:`Start` button is clicked again, a pop-up warning appears. The pop-up warning instructs
+   the user to click an :guilabel:`Update` button to apply modifications to the campaign that have
+   been made.
+
+   .. image:: testing_running/workflow-modification-warning.png
+      :align: center
+      :alt: The workflow has been modified warning pop-up window of a marketing campaign form.
+
+   Be aware that participants that had no more activities left to recieve/interact with in the
+   campaign, **can** be reintroduced into the campaign, and new traces could be created for them.
```
Comment (tiku-odoo) [inline content/applications/marketing/marketing_automation/getting_started/testing_running.rst:105]: What does this mean in layman's terms? This is a repeat of what the screenshot says.

File: content/applications/marketing/marketing_automation/getting_started/testing_running.rst [hunk 8]
```diff
@@ -43,10 +55,83 @@ To start a test, click the :guilabel:`Run` button, represented by a :guilabel:`
...
+.. image:: testing_running/campaign-smart-buttons.png
+   :align: center
+   :alt: The row of smart buttons that appear in a currently running marketing campaign in Odoo.
+
+Stopping campaigns
+==================
+
+To stop a campaign that's currently running, navigate to the :menuselection:`Marketing Automation
+app`, and select the desired campaign to stop. On the campaign detail form, click the
+:guilabel:`Stop` button in the upper-left corner.
```
Comment (tiku-odoo) [inline content/applications/marketing/marketing_automation/getting_started/testing_running.rst:121]:
```suggestion
To stop a campaign that's currently running, navigate to the :menuselection:`Marketing Automation
app`, and select the desired campaign to stop. On the campaign detail form, click the
:guilabel:`Stop` button in the upper-left corner.
```
Again this is the mention of the app not the application

File: content/applications/marketing/marketing_automation/getting_started/testing_running.rst [hunk 9]
```diff
@@ -2,33 +2,45 @@
...
+=================
 
-On the campaign detail form, make sure the campaign already has activities configured (or build a
-campaign by following the directions here on :doc:`this documentation
+To test a marketing campaign, open the :menuselection:`Marketing Automation` application, and select
+the desired campaign to test, which reveals that campaign's detail form.
+
+On the campaign detail form, make sure the campaign already has activities configured in the
+workflow (or build a campaign by following the directions on :doc:`this documentation
 </applications/marketing/marketing_automation/getting_started/workflow_activities>`).
```
Comment (tiku-odoo) [inline content/applications/marketing/marketing_automation/getting_started/testing_running.rst:18]:
```suggestion
.. seealso::
  Testing marketing automations in meant to be performed in the production version of the database.           Duplicate or trial databases will have limited email sending capabilities. 
```

File: content/applications/marketing/marketing_automation/getting_started/testing_running.rst [hunk 10]
```diff
@@ -43,10 +55,83 @@ To start a test, click the :guilabel:`Run` button, represented by a :guilabel:`
...
    :alt: Workflow test progress in Odoo Marketing Automation.
 
-Once all the workflow activities are completed, the test will end, and move to the
-:guilabel:`Completed` stage. To stop a test before all the workflow activities are completed, click
-the :guilabel:`Stop` button in the upper-left corner of the campaign test page.
+Once all the workflow activities are completed, the test ends, and the status bar (in the
+upper-right corner) moves to the :guilabel:`Completed` stage.
+
+To stop a test before all the workflow activities are completed, click the :guilabel:`Stop` button
+in the upper-left corner of the campaign test page.
```
Comment (tiku-odoo) [inline content/applications/marketing/marketing_automation/getting_started/testing_running.rst:71]: Maybe reference Stopping a campaign below?

File: content/applications/marketing/marketing_automation/getting_started/testing_running/workflow-modification-warning.png
Comment (tiku-odoo) [inline content/applications/marketing/marketing_automation/getting_started/testing_running/workflow-modification-warning.png:1]: This file is over 768 pixels wide. Should resize.

File: content/applications/marketing/marketing_automation/getting_started/testing_running/test-screen.png
Comment (tiku-odoo) [inline content/applications/marketing/marketing_automation/getting_started/testing_running/test-screen.png:1]: This file is over 768 pixels wide. Should resize.

File: content/applications/marketing/marketing_automation/getting_started/testing_running/workflow-test-progress.png
Comment (tiku-odoo) [inline content/applications/marketing/marketing_automation/getting_started/testing_running/workflow-test-progress.png:1]: This file is over 768 pixels wide. Should resize.

File: content/applications/marketing/marketing_automation/getting_started/testing_running/launch-test-popup-window.png
Comment (tiku-odoo) [inline content/applications/marketing/marketing_automation/getting_started/testing_running/launch-test-popup-window.png:1]: This file is over 768 pixels wide. Should resize.

**Review by tiku-odoo — COMMENTED** [review]
@ksc-odoo Great work on this documentation. I've checked it for functionality and structure/grammar. I've made a handful of requested changes. Overall really great work, keep it up :+1: Let me know if you have any questions!

File: content/applications/marketing/marketing_automation/getting_started/testing_running.rst (hunk 4 above)
Comment (ksc-odoo) [reply content/applications/marketing/marketing_automation/getting_started/testing_running.rst:46]: I completely understand. I added a red arrow indicator for the child activity image - I believe that'll be sufficient enough to separate the two. Great note!

File: content/applications/marketing/marketing_automation/getting_started/testing_running.rst (hunk 5 above)
Comment (ksc-odoo) [reply content/applications/marketing/marketing_automation/getting_started/testing_running.rst:77]: Great catch!

**Comment by ksc-odoo** [comment]
@tiku-odoo thanks for the awesome review! I made all your suggested edits - I think the doc is even stronger now! 👍 I think this is ready for a look from anyone else on the @odoo/marketing-automation-doc-review

**Comment by ksc-odoo** [comment]
Thanks for the super-helpful feedback @Felicious -- I made all your suggested edits, and believe this is ready for another round of peer review @meng-odoo 👍

**Review by StraubCreative — APPROVED** [review]
@robodoo r+
